controller/cache/cluster.go

```diff
@@ -21,12 +21,17 @@ import (
 )
 
 const (
-	clusterSyncTimeout  = 1 * time.Hour
+	clusterSyncTimeout  = 24 * time.Hour
```

Comment:
accidental?

Reply:
This is intentional. I want re-sync cluster cache less often since now controller should never miss watch events.